src/App.js

```diff
@@ -13,27 +13,22 @@ class App extends Component {
       error: '',
     }
   }
-  componentDidMount(event) {
-    fetchData
-      .getAllMovies()
+  componentDidMount() {
+    fetchData.getAllMovies()
       .then(data => {
         this.setState({moviesKey: data.movies})
       })
       .catch(error => this.setState({error: 'uh oh'}))
   }
 
   showMovie = id => {
```

Comment:
`showMovie` is an interesting name - this function sets the movies to state, rather than displaying them. Maybe selectMovie makes more sense?